bin/notificationSubscribedUsers.ts

```diff
@@ -0,0 +1,9028 @@
+import { UserAction, UserActionType } from '../src/entity';
+import createOrGetConnection from '../src/db';
+
+const subscribedUsers = [
```

Comment:
Please remove the array from this file. It should be provided as a runtime argument.
I don't want to expose user ids in our codebase

Reply:
Oh, right. How would we receive the values and what would be the type of the argument? Will it be a comma-separated string?

Reply:
I think a path to a file and then we can read the file in runtime